### ISSUE TITLE
Incorrectly reports GlueDataBrew/Properties/Steps/Action/Paremters

### ISSUE THREAD

**Issue by ueberhund** [aws-cloudformation/cfn-python-lint#1869]
This occurs with version 0.44.3. Given this cfn:

```yaml
DataBrewRecipe:
    Type: AWS::DataBrew::Recipe
    Properties:
      Name: !Sub ${AWS::StackName}-recipe
      Steps:
      - Action:
          Operation: SPLIT_COLUMN_SINGLE_DELIMITER
          Parameters:
            IncludeInSplit: 'true'
            Limit: '1'
            Pattern: '-'
            SourceColumn: time-tag
      - Action:
          Operation: RENAME
          Parameters:
            SourceColumn: time-tag_1
            TargetColumn: year
      - Action:
          Operation: RENAME
          Parameters:
            SourceColumn: time-tag_2
            TargetColumn: month
```

cfn-lint reports `E3002 Property is an object instead of String` for each of the Parameters. However, per [the cfn documentation](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-databrew-recipe.html), this is how to correctly define a Glue DataBrew recipe.

**Comment by kddejong** [comment]
The [spec](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-databrew-recipe-action.html#cfn-databrew-recipe-action-parameters) is a little off on this one.  Not sure why this isn't translating yet.  Looking into it.

**Comment by kddejong** [comment]
@PatMyron are we able to handle `anyOf`.

https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-databrew/blob/main/aws-databrew-recipe/aws-databrew-recipe.json#L125-L133
